README-hi.md

```diff
@@ -1,85 +1,31 @@
-
-कुबरनेट्स प्रलेखन में योगदान देने के बारे में अधिक जानकारी के लिए, देखें:
-
-* [योगदान देना शुरू करें](https://kubernetes.io/docs/contribute/start/)
-* [परिवर्तनों को अंतिम चरण में लेजाएं](http://kubernetes.io/docs/contribute/intermediate#view-your-changes-locally)
-* [पेज टेम्पलेट](https://kubernetes.io/docs/contribute/style/page-content-types/)
-* [प्रलेखन शैली गाइड](http://kubernetes.io/docs/contribute/style/style-guide/)
-* [स्थानीयकरण कुबरनेट्स प्रलेखन](https://kubernetes.io/docs/contribute/localization/)
-
-## `README.md`'s स्थानीयकरण कुबरनेट्स प्रलेखन
-
-आप हिंदी स्थानीयकरण के मैन्टेनरों तक पहुँच सकते हैं:
-
-* Anubhav Vardhan ([Slack](https://kubernetes.slack.com/archives/D0261C0A3R8), [Twitter](https://twitter.com/anubha_v_ardhan), [GitHub](https://github.com/anubha-v-ardhan))
-* Divya Mohan ([Slack](https://kubernetes.slack.com/archives/D027R7BE804), [Twitter](https://twitter.com/Divya_Mohan02), [GitHub](https://github.com/divya-mohan0209))
-* Yashu Mittal ([Twitter](https://twitter.com/mittalyashu77), [GitHub](https://github.com/mittalyashu))
-
-* [Slack channel](https://kubernetes.slack.com/messages/kubernetes-docs-hi)
-
-## स्थानीय रूप से डॉकर का उपयोग करके साइट चलाना
+यहाँ आपके दिए गए कंटेंट पर आधारित एक सरल हिंदी README फ़ाइल है:
 
-कुबरनेट्स वेबसाइट को स्थानीय रूप से चलाने के लिए अनुशंसित तरीका एक विशेष [डॉकर](https://docker.com) image को चलाना है, जिसमें [Hugo](https://gohugo.io) स्टेटिक साइट जनरेटर शामिल है।
+---
 
-> यदि आप विंडोज पर हैं, तो आपको कुछ और टूल्स की आवश्यकता होगी जिन्हें आप [Chocolatey](https://chocolatey.org) के साथ इंस्टॉल कर सकते हैं।
-
-यदि आप डॉकर के बिना स्थानीय रूप से वेबसाइट चलाना पसंद करते हैं, तो नीचे दिए गए Hugo का उपयोग करके स्थानीय रूप से [साइट को चलाने](#hugo-का-उपयोग-करते-हुए-स्थानीय-रूप-से-साइट-चलाना) का तरीका देखें।
-
-यदि आप [डॉकर](https://www.docker.com/get-started) चला रहे हैं, तो स्थानीय रूप से `कुबेरनेट्स-ह्यूगो` Docker image बनाएँ:
-
-```bash
-make container-image
-```
-
-एक बार image बन जाने के बाद, आप साइट को स्थानीय रूप से चला सकते हैं:
-
-```bash
-make container-serve
-```
-
-साइट देखने के लिए अपने browser को `http://localhost:1313` पर खोलें। जैसा कि आप source फ़ाइलों में परिवर्तन करते हैं, Hugo साइट को अपडेट करता है और browser को refresh करने पर मजबूर करता है।
-
-## Hugo का उपयोग करते हुए स्थानीय रूप से साइट चलाना
-
-Hugo निर्देशों के लिए [आधिकारिक Hugo प्रलेखन](https://gohugo.io/getting-started/installing/) देखें। [`Netlify.toml`](netlify.toml#L9) फ़ाइल में `HUGO_VERSION` environment variable द्वारा निर्दिष्ट Hugo version को install करना सुनिश्चित करें।
-
-जब आप Hugo को install करते हैं तो स्थानीय रूप से साइट को चलाने के लिए:
-
-```bash
-make serve
-```
-
-यह पोर्ट `1313` पर Hugo सर्वर को शुरू करेगा। साइट देखने के लिए अपने browser को `http://localhost:1313` पर खोलें। जैसा कि आप source फ़ाइलों में परिवर्तन करते हैं, Hugo साइट को अपडेट करता है और एक browser को refresh करने पर मजबूर करता है।
```

Comment:
@RajathKP, any reason to remove this content, does not seem like it has any issues.